core/src/main/java/io/aiven/klaw/service/HAMessagingServiceI.java

```diff
@@ -0,0 +1,8 @@
+package io.aiven.klaw.service;
+
+public interface HAMessagingServiceI<T> {
```

Comment:
do we want to keep interfaces in a sub dir ?

Reply:
I like that yes